docs/userguide/containers/dockerrepos.md

```diff
@@ -42,6 +42,7 @@ You can now commit and push your own images up to your repos on Docker Hub.
 > **Note:**
 > Your authentication credentials will be stored in the `~/.docker/config.json`
 > authentication file in your home directory.
+> For Windows users, this Docker file is stored in `$HOME/.docker/config.json`
```

Comment:
@moxiegirl I updated this PR from the gist. Do you want to review that it is exactly as you intended?